src/protocol/prf_sharding/mod.rs

```diff
@@ -557,22 +540,41 @@ where
 /// another secret-shared bit indicating if a given row is within the attribution window. We multiply these two bits together and
 /// multiply it with the bits of the `trigger_value` in order to zero out contributions from unattributed trigger events.
 ///
+#[allow(clippy::too_many_arguments)]
```

Comment:
`too_many_lines` is no more, but now `too_many_arguments`... (8 vs. 7)
I could create a struct for timestamps + attribution window, but that requires either clone()'s or a life-time. I think this is more readable.